test/lit.cfg

```diff
@@ -374,6 +375,7 @@ config.substitutions.append( ('%llvm-dis', config.llvm_dis) )
 config.substitutions.append( ('%swift-demangle-yamldump', config.swift_demangle_yamldump) )
 config.substitutions.append( ('%Benchmark_O', config.benchmark_o) )
 config.substitutions.append( ('%Benchmark_Driver', config.benchmark_driver) )
+config.substitutions.append( ('%llvm-strings', config.llvm_strings) )
```

Comment:
Nitpick: Please *don't* add a substitution for this. The LLVM tools are already in the search path, so you can just say `llvm-strings`.